projects/1.16/assets/xaeros-minimap/xaerobetterpvp/lang/zh_cn.json

```
@@ -1,354 +1,552 @@
+	"gui.xaero_always_show_distance": "总是显示路径点距离。",
+	"gui.xaero_waypoints_distance_min": "最小路径点绘制距离。",
+	"gui.xaero_box_entity_depth": "根据实体相对于您的Y轴高度，显示较暗的圆点。",
+	"gui.xaero_box_arrow_scale": "小地图方向固定和其他一些情况下使用的箭头比例。",
+	"gui.xaero_box_distance": "可以显示到目的地距离的路径点。",
+	"gui.xaero_box_cave_maps": "绘图模式可以深入到比地表更深的地方，主要是显示地下洞穴和建筑内部。屋顶大小代表一个实心的水平“正方形”的大小，需要在你的上方检测到它才能激活洞穴模式。",
+	"gui.xaero_sort_symbol": "开始",
+	"gui.xaero_sort_unsorted": "未分类",
+	"gui.xaero_sort": "分类",
+	"gui.xaero_centered_enlarged": "放大时居中",
+	"gui.xaero_world_teleport_command": "传送聊天命令",
+	"gui.xaero_teleportation_tooltip": "一旦禁用，只能通过配置文件启用！",
+	"gui.xaero_waypoint_distance_vertical_visibility_angle": "WP距离、水平、角度",
+	"gui.xaero_use_multiworld": "多维度检测",
+	"gui.xaero_biomes_vanilla": "原版模式生物群系",
+	"gui.xaero_restore": "恢复",
+	"gui.xaero_temporary": "暂时的",
+	"gui.xaero_12h": "12h",
+	"gui.xaero_24h": "24h",
+	"gui.xaero_show_time": "显示游戏时间",
+	"gui.xaero_unreachable_dimension": "无法访问目标维度。",
+	"gui.xaero_cross_tp": "跨维度TP",
+	"gui.xaero_mp_cross_tp": "MP跨维度TP",
+	"gui.xaero_visit_needed": "§e您需要先访问此维度以将其转换为新格式！",
+	"gui.xaero_render_layer": "覆盖渲染层",
+	"gui.xaero_set_name": "设置名称",
+	"gui.xaero_display_all_sets": "切换所有WP设置渲染",
+	"gui.xaero_down": "[向下]",
+	"gui.xaero_show_light_level": "显示光照等级",
+	"gui.xaero_partial": "元素",
+	"gui.xaero_roof": "顶部",
+	"gui.xaero_share_msg2": "请确保你没有暴露一个秘密地点。",
+	"gui.xaero_share": "分享",
+	"gui.xaero_divide_msg2": "你确定要将所有子世界坐标除以8吗？",
+	"gui.xaero_multiply_msg1": "此选项仅应用于修复错误的路径点坐标。",
+	"gui.xaero_divide_all_by_8": "子世界坐标 / 8",
+	"gui.xaero_multiply_all_by_8": "子世界坐标 * 8",
+	"gui.xaero_item_tooltip_time": "显示为",
+	"gui.xaero_entity_tooltip_settings": "物品提示文本设置",
+	"gui.xaero_item_tooltip": "物品提示文本",
+	"gui.xaero_delete_multi_world_msg2": "此行动将无法复原。请确定自己了解现在所进行的动作。",
+	"gui.xaero_delete_multi_world": "删除子世界",
+	"gui.xaero_delete_world_msg2": "此行动将无法复原。请确定自己了解现在所进行的动作。",
+	"gui.xaero_delete_world": "删除世界/服务器",
+	"gui.xaero_make_multi_automatic_msg2": "这将交换所选子世界和自动子世界的所有路径点数据，从而模拟使所选子世界自动化。请确保清楚你在做什么。",
+	"gui.xaero_make_multi_automatic_msg1": "你确定？",
+	"gui.xaero_make_automatic_msg2": "这将交换所选世界/服务器和自动世界/服务器的所有路径点数据，从而模拟使所选世界自动化。请确保清楚你在做什么。",
+	"gui.xaero_make_automatic_msg1": "你确定？",
+	"gui.xaero_options": "选项",
+	"gui.xaero_open_slime": "按键摁下时打开",
+	"gui.xaero_hide_world_names": "隐藏世界名称/IP",
+	"gui.xaero_waypoint_opacity_ingame": "游戏内路径点透明度",
+	"gui.xaero_subworld_dimension": "子世界/维度",
+	"gui.xaero_team": "团队",
+	"gui.xaero_block_transparency": "方块透明度",
+	"gui.xaero_terrain_depth": "地形深度",
+	"gui.xaero_entity_info_potion_effects": "药水效果（仅限单人游戏）",
+	"gui.xaero_entity_radar": "实体雷达",
+	"gui.xaero_patron_capes": "Patreon披风",
+	"gui.xaero_use_world_map": "使用世界地图区块",
+	"gui.xaero_world_map": "世界地图",
+	"gui.xaero_safe_mode_box": "使用基于Java（CPU）的等效模组而不是OpenGL（GPU）。以防正常模式失败，安全模式仅作为B计划。并非所有功能都在安全模式下工作。",
+	"gui.xaero_smooth_dots": "平滑点",
+	"gui.xaero_arrow_colour": "箭头颜色",
+	"gui.xaero_entity_info_armour": "护甲值",
+	"gui.xaero_entity_info_armour_numbers": "护甲数量",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"gui.xaero_entity_info_armour_numbers": "盔甲数量",
```